src/components/TaskDashboard.tsx

```diff
@@ -16,21 +17,53 @@ function TaskDashboard({
   editingIdValue,
   startEditing,
   editTask,
+  handleCheck,
 }: TaskDashboardProps) {
+  const [taskBoardRender, setTaskBoardRender] = useState([...taskList]);
```

Comment:
tinha te falado naquela hora mais pra testar , acho q n presisa de dar spread aq n
```suggestion
  const [taskBoardRender, setTaskBoardRender] = useState(taskList);
```

Reply:
So complementando, spread operator é bom mas é uma operação pesada. Evite usar quando não for necessário.